spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -332,18 +332,13 @@ def main(argv: Sequence[str]):
         smooth=arguments.centerline_smooth)
     # registration parameters
     if arguments.param is not None:
-        # reset parameters but keep step=0 (might be overwritten if user specified step=0)
+        # reset parameters but keep step=0 (can be overwritten if user specified step=0)
```

Comment:
Re: This comment... In what cases would the user want to overwrite `step=0`? Do we want to test this case?

For example, what if the user specifies a different `dof`? Would that cause a crash? (Or, is this such a rare thing for the user to do that the user should basically expect crashes/unstable behavior from modifying step 0?)

Reply:
> Re: This comment... In what cases would the user want to overwrite step=0? Do we want to test this case?

ideally we should. For subsequent PR though

> For example, what if the user specifies a different dof? Would that cause a crash? (Or, is this such a rare thing for the user to do that the user should basically expect crashes/unstable behavior from modifying step 0?)

hum... hopefully not 😬